vmdb/spec/models/miq_request_spec.rb

```diff
@@ -291,4 +291,38 @@
       expect(provision_request.approval_state).to eq('denied')
     end
   end
+
+  context '#post_create_request_tasks' do
+    context 'VM provisioning' do
+      let(:description) { 'my original information' }
+      let(:template)    { FactoryGirl.create(:template_vmware, :ext_management_system => FactoryGirl.create(:ems_vmware_with_authentication)) }
+      let(:request)     { FactoryGirl.create(:miq_provision_request, :userid => fred.userid, :description => description, :src_vm_id => template.id) }
```

Comment:
You can use FactoryGirl.build on the request since it does not need to be saved to the db.